packages/@lwc/integration-karma/test/synthetic-shadow/active-element/index.spec.js

```diff
@@ -0,0 +1,34 @@
+import { createElement } from 'lwc';
+import Source from 'x/source';
+import Target from 'x/target';
+
+if (!process.env.NATIVE_SHADOW) {
+    describe('activeElement', () => {
+        it('can call shadowRoot.activeElement on transported node with no lwc:dom=manual', async () => {
+            const source = createElement('x-source', { is: Source });
+            const target = createElement('x-target', { is: Target });
+            document.body.appendChild(source);
+            document.body.appendChild(target);
+            await Promise.resolve();
+
+            const button = source.shadowRoot.querySelector('button');
+
+            // append directly to the element, not to some <div lwc:dom=manual> inside of it as recommended
+            target.appendChild(button);
+
+            // make active
+            button.focus();
+
+            let activeElement;
+
+            expect(() => {
+                activeElement = target.shadowRoot.activeElement;
+            }).toLogErrorDev(
+                /NodeOwnedBy\(\) should never be called with a node that is not a child node of/
+            );
```

Comment:
I only managed to find a repro for one of the six log lines. But I figure it's pretty good to at least have one test here.